src/server/controllers/UserService.ts

```diff
@@ -53,8 +53,21 @@ export class UserService {
     public static create(user: IUserModel): Promise<IUserModel | undefined> {
         // TODO: user validation
 
-        return query(`INSERT INTO users(email, name, userClass, hash, salt)
-                        VALUES('${user.email}', '${user.name}', ${user.userClass}, '', '')`)
+        let insertInto = '';
+        let value = '';
+        if (user.github) {
+            insertInto = 'github';
+            value = user.github;
+        } else if (user.facebook) {
+            insertInto = 'facebook';
+            value = user.facebook;
+        } else if (user.google) {
+            insertInto = 'google';
+            value = user.google;
+        }
+
+        return query(`INSERT INTO users(email, name, userClass, hash, salt, ${insertInto})
```

Comment:
agreed re: sql injection, I think even if we're switching to an ORM soon it's worth switching these to prepared statements asap

Reply:
#21